components/sts/org.wso2.carbon.identity.sts.passive.ui/src/main/java/org/wso2/carbon/identity/sts/passive/ui/PassiveSTS.java

```diff
@@ -441,6 +444,10 @@ private void sendFrameworkForLogout(HttpServletRequest request, HttpServletRespo
         authenticationRequest.setRequestQueryParams(request.getParameterMap());
         authenticationRequest.setCommonAuthCallerPath(selfPath);
         authenticationRequest.appendRequestQueryParams(request.getParameterMap());
+        // According to ws-federation-1.2-spec; 'wtrealm' will not be sent in the Passive STS Logout Request.
+        if (sessionDTO.getRealm() == null || sessionDTO.getRealm().trim().length() == 0) {
+            authenticationRequest.setRelyingParty(" ");
+        }
```

Comment:
Is this correct ? In the comment and condition you say realm but you are setting relyingParty